Seriously? `m_remove_addon_user_data_and_settings`? :rocket:
I think you can squeeze in a few more underscores for good measure, heheh. (yes, I saw the comment above.)

On a more serious note, bots aren't happy.

> /home/travis/build/xbmc/xbmc/xbmc/addons/AddonInstaller.cpp:808:21: error: class â€˜CAddonUnInstallJobâ€™ does not have any field named â€˜m_remove_addon_userdata_and_settingsâ€™
   : m_addon(addon), m_remove_addon_userdata_and_settings(removeData)

--- a/xbmc/addons/AddonInstaller.cpp
+++ b/xbmc/addons/AddonInstaller.cpp
@@ -805,7 +805,7 @@ void CAddonInstallJob::ReportInstallError(const std::string& addonID, const std:
 }
 
 CAddonUnInstallJob::CAddonUnInstallJob(const AddonPtr &addon, bool removeData)
-  : m_addon(addon), m_remove_addon_userdata_and_settings(removeData)
+  : m_addon(addon), m_removeData(removeData)
 { }
 
 bool CAddonUnInstallJob::DoWork()
@@ -828,7 +828,7 @@ bool CAddonUnInstallJob::DoWork()
   }
 
   ClearFavourites();
-  if (m_remove_addon_user_data_and_settings)
+  if (m_removeData)
     CFileUtils::DeleteItem("special://profile/addon_data/"+m_addon->ID()+"/", true);
 
   AddonPtr addon;

--- a/xbmc/addons/AddonInstaller.h
+++ b/xbmc/addons/AddonInstaller.h
@@ -199,5 +199,5 @@ private:
   void ClearFavourites();
 
   ADDON::AddonPtr m_addon;
-  bool m_remove_addon_user_data_and_settings;
+  bool m_removeData;
 };

